rspec_test/solver_rspec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rspec'
```

Comment:
- [ ] Great job, writing tests for all the cases in the project requirement :clap: However there is an error when trying to run the tests(see image).
  **HIT:** Kindly add  the last version of rspec gem to your gemfile ` gem 'rspec', '~> 3.4` or simply remove this line from your spec file.
![Screenshot from 2023-10-12 17-21-10](https://github.com/noelfoka/library-school/assets/91142432/3d2cccb7-9b18-4b44-9b8f-cee8454c5d92)